### ISSUE TITLE
Plot function requires igraph / ggraph?

### ISSUE THREAD

**Issue by florianhartig** [citoverse/cito#24]
Sind einige Funktionen die Abhängigkeiten haben die nicht direkt installiert werden, oder?

**Comment by ChristianAmes** [comment]
Genau, die plot funktion funktioniert nur mit igraph & ggraph (ggraph installiert igraph mit) . 
Ist also nur in 'Suggests' in der DESCRIPTION datei vorhanden. Sollte man das ändern auf Imports ? 
Die Idee dahinter ist, so wenig wie möglich packages zu installieren um die Grundfunktionalität zu erreichen.

**Comment by florianhartig** [comment]
Suggest isr eigentlich richtig aber damit sollte sie ja installiert werden

Hattest du bei install dependencies = F?

**Comment by ChristianAmes** [comment]
Ich hatte die defaluts, also NA damit sollte eigentlich suggests auch installiert werden.
Habe ein bisschen was getestet und es scheint jedoch nicht zu passieren

Bei 
> remove.packages("ggraph")
> remove.packages("cito")
> devtools::install_github("citoverse/cito") 
> requireNamespace("ggraph")

kommt: 

> Loading required namespace: ggraph
Failed with error:  ‘there is no package called ‘ggraph’’
